src/test/java/io/github/fablabsmc/fablabs/api/fiber/v1/annotation/AnnotatedSettingsTest.java

```diff
@@ -29,6 +22,8 @@
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
+import static org.junit.jupiter.api.Assertions.*;
+
```

Comment:
Won’t IDEA’s “optimize imports” macro get rid of this?

Reply:
It doesn't for me. This import was generated by IDEA, actually.